### PR TITLE
Fixes #569. Adds mob_db.txt documentation.

### LINKED ISSUE #569: add doc/mob_db.txt plis
https://github.com/rathena/rathena/blob/master/doc/mob_db.txt

### PR DESCRIPTION
**Pull request by iRagno**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** `master`

[//]: # (Master? Slave?)

**Issues addressed:** #569 add doc/mob_db.txt plis

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

None

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

---
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1623)

### CODE REVIEW

File: doc/mob_db.txt [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+		Agi: agility                  (int, defaults to 0)
+		Vit: vitality                 (int, defaults to 0)
+		Int: intelligence             (int, defaults to 0)
+		Dex: dexterity                (int, defaults to 0)
+		Luk: luck                     (int, defaults to 0)
+	}
+	ViewRange: view range                 (int, defaults to 1)
+	ChaseRange: chase range               (int, defaults to 1)
+	Size: size                            (int, defaults to 1)
+	Race: race                            (int, defaults to 0)
```
Comment (dastgir) [inline doc/mob_db.txt]: String

I see some inconsistency on mob_db.conf too, maybe update that too?

File: doc/mob_db.txt [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+		Str: strength                 (int, defaults to 0)
+		Agi: agility                  (int, defaults to 0)
+		Vit: vitality                 (int, defaults to 0)
+		Int: intelligence             (int, defaults to 0)
+		Dex: dexterity                (int, defaults to 0)
+		Luk: luck                     (int, defaults to 0)
+	}
+	ViewRange: view range                 (int, defaults to 1)
+	ChaseRange: chase range               (int, defaults to 1)
+	Size: size                            (int, defaults to 1)
```
Comment (dastgir) [inline doc/mob_db.txt]: String

**Comment by iRagno** [comment]
Thanks for review @dastgir. Applied requested changes.

**Comment by iRagno** [comment]
The only field I couldn't explain properly was DamageMotion, about how it's calculated from official and what that does.

And I have a question regarded to monsters aspd values. I noticed a way to convert from "official" aspd values to emulation, by dividing 1000 / aspd.

In example, there is Desert Wolf wich has 2.38 aspd according to http://www.divine-pride.net/database/monster/1106 

1000 / 2.38 would be 420.1681 and that is the AttackMotion value. However, this Desert Wolf will not be able to do 2.38 attacks per second because AttackDelay is 1120, and with that value will do 0.89 hits per second.

That is because Desert Wolf has an animation motion delay that allows him to do 2.38 attacks per second, but has an AttackDelay that limits him to do only 0.89 hits per second, and maybe with buffs may reach 2.38 aspd, that should be the correct, as I understand.

What is your opinion about this? Did I understand something wrong?

**Comment by Ridley8819** [comment]
Thank you Ragno

**Pull request merged**